flaskapp/view_functions/ivrflow.py

```diff
@@ -29,23 +29,25 @@ def voice_joined():
     else:
         resp.say(f'We got your answer {answer}. We hope you will back us later. Take care.')
         resp.hangup()
-    return(str(resp))
+    return (str(resp))
```

Comment:
It definitely is not needed to put extra parentheses when returning a value: `return str(resp)` would be ok.

Reply:
done